### PR TITLE
update PushTraceSpans to return dropped spans instead of good spans.

### LINKED ISSUE #250: PushTraceSpans should return dropped spans instead of good spans
This is to make it consistent with PushMetricsProto()

### PR DESCRIPTION
**Pull request by rghetia**
fixes #250

### CODE REVIEW

**Comment by rghetia** [comment]
@ibawt PTAL.

File: trace.go [hunk 1]
```diff
@@ -148,7 +148,8 @@ func (e *traceExporter) pushTraceSpans(ctx context.Context, node *commonpb.Node,
 	ctx, cancel := newContextWithTimeout(ctx, e.o.Timeout)
 	defer cancel()
 
-	return e.client.BatchWriteSpans(ctx, &req)
+	err := e.client.BatchWriteSpans(ctx, &req)
+	return 0, err
```
Comment (dinooliva) [inline trace.go]: Rhere will never be any dropped spans for stackdriver, is that correct?

Comment (songy23) [reply trace.go]: I think if `err` is not `nil`, we should return `len(protoSpans)` here.

Comment (rghetia) [reply trace.go]: @dinooliva it will be all or none.

@songy23 fixed it.

**Pull request merged**